model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -246,13 +246,13 @@ object Workspace {
     )
   }
 
-  def apply(namespace: String,
-            name: String,
-            workspaceId: String,
-            createdDate: DateTime,
-            lastModified: DateTime,
-            createdBy: String,
-            attributes: AttributeMap
+  def buildMcWorkspace(namespace: String,
```

Comment:
I was getting confused why workspace objects being instantiated via the apply() method were getting a type of `mc` in the DB; this hopefully makes it more explicit and less prone to misuse.